src/components/TextFileLink/index.android.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import RNFetchBlob from 'react-native-blob-util';
+import * as FileUtils from '../../libs/fileDownload/FileUtils';
+
+const propTypes = {
+    fileName: PropTypes.string,
+    textContent: PropTypes.string,
+    children: PropTypes.func,
+};
+
+const defaultProps = {
+    fileName: '',
+    textContent: '',
+    children: () => {},
+};
+
+const TextFileLink = (props) => {
```

Comment:
Agree. What do you think of renaming it to `ButtonFileLink` and making it always return a `Button`?

Reply:
I agree with the thoughts - a functional component not rendering anything looks weird.
We can keep it is a simple function. I like that idea, but not sure where we'll have that function.
Making a ButtonLink component also looks like a good idea, but I wonder, we'll then be limited to have download functionality in other components.

Reply:
If we go with the function approach, we could probably do something similar to `libs/fileDownload` because we will need different implementations for web, ios and android. And maybe call it `libs/localFileDownload`, since this is only for local files. wdyt?

Reply:
Replaced `TextFileLink` with a simple function on `/libs/localFileDownload`